test/validation-ui-tests.js

```diff
@@ -625,6 +625,74 @@ QUnit.test('Issue #44 - Validation Element - Is Invalid Test', function(assert)
 
 });
 
+QUnit.test('Validatable can become non-validatable', 0, function(assert) {
+    var vm = {
+        testObj: ko.observable().extend({
+            required: { params: true, message: "oh" }
+        })
+    };
+
+    addTestHtml('<input type="text" id="testElement" data-bind="value: testObj, validationElement: testObj, validationMessage: testObj"/>');
+
+    // make sure we allow element decorations
+    ko.validation.init({ decorateInputElement: true }, true);
+
+    applyTestBindings(vm);
+
+    vm.testObj.extend({validatable: false});
+});
+
+QUnit.test('Issue #519 - validationMessage and validationElement can be applied to non-validatable element', 0, function(assert) {
```

Comment:
It would be better to make some assertions related to the markup, eg: the element with `validationMessage` is hidden, has no content; the input doesn't get decorated with `validationElement` class.